src/md/src/Analysis/AutoAnalysisOfLaunches.md

```diff
@@ -42,90 +40,249 @@ The following info is sent:
 * A launch name;
 * Unique ID;
 
-So that the more you analyze, the more data in the ElasticSearch, the more accurate results you will get in the end of the process. 
+Service Analyzer forms a query with following data to the  service ElasticSearch.:
```

Comment:
composes a search query to ElasticSearch that consists of the following data fields: